dotnet/src/SemanticKernel.Abstractions/TemplateEngine/IPromptTemplateEngine.cs

```diff
@@ -13,16 +11,6 @@ namespace Microsoft.SemanticKernel.TemplateEngine;
 /// </summary>
 public interface IPromptTemplateEngine
 {
-    /// <summary>
-    /// Given a prompt template string, extract all the blocks (text, variables, function calls)
-    /// </summary>
-    /// <param name="templateText">Prompt template (see skprompt.txt files)</param>
-    /// <param name="validate">Whether to validate the blocks syntax, or just return the blocks found, which could contain invalid code</param>
-    /// <returns>A list of all the blocks, ie the template tokenized in text, variables and function calls</returns>
-    IList<Block> ExtractBlocks(
```

Comment:
Can we keep this? It could be useful on the extension side for highlighting user and SDK/AI provided values in the prompt preview. Determining these values using a diff didn't always provide an accurate result and I think we shouldn't need to re-calculate this again on the extension side.